src/main/java/pages/checkoutConfirmationPage/OrderSummaryTable.java

```diff
@@ -5,23 +5,53 @@
 
 import java.math.BigDecimal;
 
-public class OrderSummaryTable {
+import static engine.drivers.WebDriverFactory.getWebDriverInstance;
+import static enums.Currency.convertCurrencyFromCookie;
 
+public class OrderSummaryTable {
```

Comment:
Also this one? We need that?

Reply:
Prices are also found here and a refactor was needed to use ``convertCurrencyFromCookie(priceText);``.